g3/git/shell.py

```diff
@@ -0,0 +1,237 @@
+import asyncio
```

Comment:
I am working on a `client.py` module for `generate` and `github` packages. Shall we name this module/class `client.py/Client` too in favour of consistency?

Reply:
Sure we can transfer it later. I will merge this for now and I'll open a separate PR to refactor